pages/common/dot.md

```diff
@@ -1,12 +1,25 @@
 # dot
 
-> A command-line tool to produce layered drawings of directed graphs.
-> More information: <https://www.graphviz.org/pdf/dotguide.pdf>.
+> Render an image of a _linear directed_ network graph from a `graphviz` file.
+> Layouts are `dot`, `neato`, `twopi`, `circo`, `fdp`, `sfdp`, `osage` & `patchwork`.
+> More information: <http://graphviz.org/doc/info/command.html>.
 
-- Render an image file and determine output filename based on input filename and selected format:
+- Render a 'png' image with a filename based on the input filename and output format:
```

Comment:
```suggestion
- Render a `.png` image with a filename based on the input filename and output format:
```
The dot is optional
Also, if this example shows how to render a `png` image, it shouldn't say `based on the output format`.

Reply:
Disagree. `png` is used in both the file extension and the -Tpng flag. There is no dot in the flag.

Disagree. There isn't enough room on one line to explain that you can use any of 10 formats AND that the -O flag will build a filename from the input and use the format for the output file extension, so that information is introduced slowly. This example explains that -O will construct an appropriate output filename and the third example introduces all the other possible format and uses -O again like the initial specific example.

Reply:
I'm still a bit confused, but I'm not the one who knows the command. However, we should still replace the backtick:
```suggestion
- Render a `png` image with a filename based on the input filename and output format:
```

Reply:
Hmm. I thought that I had. Got it now.